### PR TITLE
IBM - CI/CD Pipeline Setup

### PR DESCRIPTION
**Pull request by VGmt**

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -25,6 +25,14 @@
 import logging
 from enum import Enum
 from flask_sqlalchemy import SQLAlchemy
+from retry import retry
+from requests import HTTPError, ConnectionError
```
Comment (biegel) [inline service/models.py:30]: i think you need an ```import os``` here

File: service/models.py [hunk 2]
```diff
@@ -72,6 +81,14 @@ class Wishlist(db.Model):
...
         return "<Wishlist %r id=[%s]>" % (self.name, self.id)
 
+    @retry(
+        HTTPError,
+        delay=RETRY_DELAY,
+        backoff=RETRY_BACKOFF,
+        tries=RETRY_COUNT,
+        logger=logger,
+    )
+
```
Comment (billykern) [inline service/models.py]: i think the convention is not to leave a line between a decorator and the method/function being decorated, please remove the blank lines

File: service/models.py [hunk 3]
```diff
@@ -145,12 +178,30 @@ def init_db(cls, app):
...
+    @classmethod
+    @retry(
+        HTTPError,
+        delay=RETRY_DELAY,
+        backoff=RETRY_BACKOFF,
+        tries=RETRY_COUNT,
+        logger=logger,
+    )
+
     @classmethod
```
Comment (billykern) [inline service/models.py]: this method still has a `@classmethod` decorator (in addition to the one being added above), please remove this one with this change

**Review by billykern — CHANGES_REQUESTED** [review]

File: service/models.py (hunk 2 above)
Comment (VGmt) [reply service/models.py]: Nice catch, thanks for the input. I'll modify it.

**Review by billykern — APPROVED** [review]

**Comment by biegel** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/c836ff1f-3847-4491-890e-32fa8a8c479d?env_id=ibm:yp:us-south "nyu-wishlist-service-sum21-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/91c3adc5-dd63-41ae-ae8c-27f47731cb24 "nyu-wishlist-service-sum21") deployed [wishlists](https://github.com/NYU-DevOps-Squad-Wishlists/wishlists/releases/tag/deploy-ibm-yp-us-south-vbg5833-nyu.edu-dev-20210810-201500 "deploy-ibm-yp-us-south-vbg5833-nyu.edu-dev-20210810-201500") to [dev](https://console.bluemix.net/apps/157d30a4-e111-43a5-96d2-fd0b17f27b1a?env_id=ibm:yp:us-south "ibm:yp:us-south:vbg5833@nyu.edu:dev"), including commit 47b0a0337946a7d923794db05b8a5db18790c65b

**Comment by biegel** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/c836ff1f-3847-4491-890e-32fa8a8c479d?env_id=ibm:yp:us-south "nyu-wishlist-service-sum21-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/91c3adc5-dd63-41ae-ae8c-27f47731cb24 "nyu-wishlist-service-sum21") deployed [wishlists](https://github.com/NYU-DevOps-Squad-Wishlists/wishlists/releases/tag/deploy-ibm-yp-us-south-vbg5833-nyu.edu-prod-20210811-010226 "deploy-ibm-yp-us-south-vbg5833-nyu.edu-prod-20210811-010226") to [prod](https://console.bluemix.net/apps/3ab92089-100f-412a-adf2-90939fc24e98?env_id=ibm:yp:us-south "ibm:yp:us-south:vbg5833@nyu.edu:prod"), including commit 47b0a0337946a7d923794db05b8a5db18790c65b

**Pull request merged**